tensorflow/go/attrs.go

```diff
@@ -198,18 +202,47 @@ func scalarAttribute(op *Operation, cname *C.char, meta C.TF_AttrMetadata) (inte
 
 	case C.TF_ATTR_SHAPE:
 		numDims := meta.total_size
+		maxElems := numDims
```

Comment:
Here too, seems like we could keep it as is and just add a case for 0?

```go
if numDims < 0 {
  return Shape{}, nil
}
if numDims == 0 {
  return ScalarShape(), nil
}
dims := make...
```

Reply:
Done.